gulp_tasks/build.js

```diff
@@ -14,6 +14,7 @@ const ngAnnotate = require('gulp-ng-annotate');
 const replace = require('gulp-replace');
 
 const conf = require('../conf/gulp.conf');
+const path = require('path');
 
 gulp.task('build', build);
```

Comment:
it should work according to the doc, but it doesn't:
```
AssertionError [ERR_ASSERTION]: Task function must be specified
```
Not too sure what I'm doing wrong?...